bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -114,6 +114,7 @@ public class ServerConfiguration extends AbstractConfiguration<ServerConfigurati
     protected static final String GC_OVERREPLICATED_LEDGER_WAIT_TIME = "gcOverreplicatedLedgerWaitTime";
     protected static final String GC_OVERREPLICATED_LEDGER_MAX_CONCURRENT_REQUESTS =
             "gcOverreplicatedLedgerMaxConcurrentRequests";
+    protected static final String GC_METADATA_OP_RATE_LIMIT = "gcMetadataOpRateLimit";
```

Comment:
Since "gc" isn't used at all in the existing rate limiting settings, this raises the question of whether `gcMetadataOpRateLimit` should have a similar name such as `compactionMetadataOpRateLimit` to be consistent with the other concepts that are used ("compaction" instead of "gc").

Reply:
This configuration affect all subtasks in gc, while compaction is just the last subtask in gc.
I think `gcMetadataOpRateLimit` is better than `compactionMetadataOpRateLimit`.